x-pack/plugins/file_upload/public/components/import_complete_view.tsx

```diff
@@ -221,8 +221,9 @@ export class ImportCompleteView extends Component<Props, {}> {
   }
 
   render() {
+    const theme = getTheme();
     return (
-      <KibanaContextProvider services={services}>
+      <KibanaContextProvider services={{ ...services, theme }}>
```

Comment:
Thanks. Nit, could you add theme to services object creation above instead of here

so change 
```
const services = {
  uiSettings: getUiSettings(),
  settings: getSettings(),
};
```

to 

```
const services = {
  uiSettings: getUiSettings(),
  settings: getSettings(),
  theme: getTheme(),
};
```

Reply:
++ That is cleaner, done! https://github.com/elastic/kibana/pull/159638/commits/21c6350b5e1aeb466a07c4c4fee18848321883f3